crates/common/types/blobs_bundle.rs

```diff
@@ -102,6 +102,14 @@ fn verify_blob_kzg_proof(
 }
 
 impl BlobsBundle {
+    pub fn empty() -> Self {
```

Comment:
I think the idiomatic way is to derive from the Default trait

Reply:
Thanks! I'll try that!